### DISCUSSION TITLE
Is this Hitsound normalization?

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22384] [Ideas]
An addition to #22367 but that one is already answered:
In some maps the mapper chooses to make the hitsounds borderline quiet at some sections to where it really throws me off rythm because I personally rely on hitsounds and the audio feedback and like them. Talking about standard here not mania btw. Can this be user controlled at some point? Ive literally broke at some point because the mapper just made one note super quiet so I thought I didnt hit it, got confused chainmissed etc. Just wish these could all be brought to a minimum loudness without destroying the "sounddesign" of the map because I dont see how adding these pitfalls to a map is helping.

Granted the times I missed notes because of this where in my earlier days but even now having lots of audio feedback and suddenly none doesnt seem fair or right for a rythm game. 

I chose the title just before posting because I remembered there is this setting and I never figured out how it works, does it solve my problem if so how does it work. Its a percentage thats all I know.

**Comment by peppy** [comment]
Yes, we do intend on adding a setting to normalise these to an average value, similar to how combo colour normalisation works.

**Reply by Inconsist** [reply]
Can you maybe elaborat on what the normalization currently does? Does it take skin hitsounds and merge them with map hitsounds somehow?

**Comment by Joehuu** [comment]
Coming from https://github.com/ppy/osu/discussions/34728 so this doesn't come as a necropost.

> I remembered there is this setting and I never figured out how it works

@Inconsist I don't think such a setting exists yet.
